### DISCUSSION TITLE
solutions for contributing to a wiki you don't own

### DISCUSSION THREAD

**Discussion by coronary** [MyHoneyBadger/awesome-github-wiki#63] [Q&A]
I wasn't able to find a straight answer but is there a solution for forking a wiki and then making a pull request to the original wiki?

if not what's the best way to suggest changes to a wiki in someone else's repo?

**Comment by alex-semenyuk** [comment]
Thanks, good question. I believe most relevant answer https://stackoverflow.com/questions/10642928/how-can-i-make-a-pull-request-for-a-wiki-page-on-github
